### PR TITLE
Print server command to events buffer

### LINKED ISSUE #476: How to know which server is in use?
As I'm setting up my eglot config, how can I check which server eglot chose for the current buffer? The events buffer doesn't have the server cmd line, nor does the *Messages* buffer. There's probably an eglot variable or function but I don't see it.

### PR DESCRIPTION
**Pull request by skangas**
Fixes #476 

* eglot.el (eglot--connect): Print server command to events
buffer.

### CODE REVIEW

**Comment by joaotavora** [comment]
Looks good and useful!  Merge away. Maybe in the cases where `saved-command` is `nil` you could still print something indicative of how the server was started/connected to.

File: eglot.el [hunk 1]
```diff
@@ -1122,6 +1123,9 @@ This docstring appeases checkdoc, that's all."
            initargs))
          (cancelled nil)
          (tag (make-symbol "connected-catch-tag")))
+    (when saved-command
+      (jsonrpc--debug server "Running language server: %s"
```
[the remark this answers is not in the document]
Comment (skangas) [reply eglot.el:1127]: Yes, I think it should probably be made external.

We currently have three calls to internal jsonrpc functions: `jsonrpc--request-continuations`, `jsonrpc--debug`, and `jsonrpc--call-deferred`.  This adds a fourth one, to `jsonrpc--debug`.

**Comment by skangas** [comment]
> Looks good and useful! Merge away.

Thanks!

> Maybe in the cases where `saved-command` is `nil` you could still print something indicative of how the server was started/connected to.

Yup, I did that.

**Comment by joaotavora** [comment]
Stefan Kangas ***@***.***> writes:

> Merged #804 into master.

Thanks!  (I've fallen slightly behind on all this newfound Eglot
mail traffic again, sorry about that)

**Pull request merged**